api/cloud_api/routes/Github.js

```diff
@@ -37,8 +39,10 @@ router.get('/getPullRequestsFromRepo', (req, res) => {
  * contributors in the past month
  */
 router.get('/getContributorsInPastMonthFromRepo', (req, res) => {
-  if(CLIENT_ID != 'NOT_SET' && CLIENT_SECRET != 'NOT_SET') {
-    return res.sendStatus(OK);
+  if(!runningInTest) {
+    if(CLIENT_ID === 'NOT_SET' && CLIENT_SECRET === 'NOT_SET') {
```

Comment:
might as well make these string checks part of the runningInTest thing

const skipHandler = !runningInTest && <insert config checks here>